samples/bluetooth/peripheral_hids_keyboard/README.rst

```diff
@@ -16,16 +17,32 @@ An LED displays the Caps Lock state, which can be modified by another connected
 This sample exposes the HID GATT Service.
 It uses a report map for a generic keyboard.
 
+You can also enable LE Secure Out-of-Band pairing with NFC feature by setting NFC_OOB_PAIRING flag in the application configuration.
+This feature is disabled by default.
+The following paragraphs highlight key differences in the application behaviour when NFC pairing is enabled.
+
+When the application starts, it initializes and starts the NFCT peripheral, which is used for pairing.
+The application does not start advertising immediately, but only when the NFC tag is read by an NFC polling device, for example a smartphone or a tablet with NFC support.
+The NDEF message that the tag sends to the NFC device contains data required to initiate pairing.
+To start the NFC data transfer, the NFC device must touch the NFC antenna that is connected to the nRF52 device.
+
+After reading the tag, the device can pair with the nRF52 device which is advertising.
+After connecting, the example application behaves the same way as the original HID Keyboard example.
+Reading the NFC tag again when the application is in a connected state causes disconnection.
+When the connection is lost, advertising does not restart automatically.
+To trigger advertising without NFC, the user can press Button 4.
```

Comment:
Move this just after the information that the advertising does not start automatically.